docs/guide/installation/production-setup.md

```diff
@@ -332,13 +343,22 @@ We're setting up the product's endpoint to https://prod.vuestorefront.io/api/cat
   },
 ```
 
-There are 27 more instances of `prod.vuestorefront.io` to be replaced with your production URL address in this file. Please just do so :)
+There are many more instances of `https://prod.vuestorefront.io/api/*` to be replaced with your production URL address in this file. Please just do so :)   
```

Comment:
I'm not sure about this information :thinking: Right now also works relative path. So we don't need to add everywhere `https://prod.vuestorefront.io`